pom.xml

```diff
@@ -104,6 +104,12 @@
 			<groupId>com.h2database</groupId>
```

Comment:
why do you still have H2 database? If it's for testing purposes, you should scope it as test.